packages/portal-api/src/ts/handler.ts

```diff
@@ -0,0 +1,34 @@
+import { addressUtils, logUtils } from '@0xproject/utils';
+import * as express from 'express';
+import * as _ from 'lodash';
+import { Op } from 'sequelize';
+
+import { models } from './models';
+
+export const handler = {
+    async getPrices(req: express.Request, res: express.Response) {
+        const tokensParam = req.query.tokens; // should be a string of comma delimited token symbols
+        const tokenSymbols = tokensParam.split(',');
+        const data = await models.Price.findAll({
+            attributes: ['symbol', 'base', 'price'],
+            where: {
+                [Op.or]: _.map(tokenSymbols, symbol => {
+                    return { symbol };
+                }),
+            },
+            order: [['timestamp', 'DESC']],
+        });
+        if (_.isEmpty(data)) {
+            res.sendStatus(404);
```

Comment:
Should this bea 404? Or just 200 with empty set returned? Usually expect 404 when the endpoint was not found.